fs/_to_file_info.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright 2018-2025 the Deno authors. MIT license.
 
 import type { FileInfo } from "./unstable_types.ts";
-import { isWindows } from "./_utils.ts";
+import { isWindows } from "../internal/_os.ts";
```

Comment:
This relative import doesn't work when it's published to JSR. (Cross package imports need to start with package name like `@std/internal/...`, but `_os.ts` is not part of public API of `@std/internal`.